war/src/main/less/base/style.less

```diff
@@ -780,7 +780,23 @@ table.parameters > tbody:hover {
 }
 
 .healthReport div.healthReportDetails {
-    display: none;
+  display: none;
+  margin-left: 20px;
+  padding: 0;
+
+  table * {
+    font-size: 0.8rem;
+    border-top: none!important;
+    border-bottom: none!important;
+  }
+
+  tbody * {
+    background: transparent!important;
+
+    &:hover {
+      background: transparent!important;
+    }
+  }
```

Comment:
The tables now have no striping anymore. It would make sense to restore that behavior as well (maybe using the same colors as the `bigtable`).

Reply:
Done :)

Reply:
i'm not seeing any striping here, (should I be?):

![image](https://user-images.githubusercontent.com/21194782/136107263-fc64a86a-3966-4b5f-af3f-b68945f2eb08.png)

Reply:
I see the striping on the build dashboard page but not the build history widget